### DISCUSSION TITLE
3-Phase usage

### DISCUSSION THREAD

**Discussion by hirani89** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#37] [Q&A] [answered]
Hi,
How would I implement this to measure power usage for my 3-phase connection. I am planning to use a single main board.
From what I understand, I could use VA and VA2 to supply separate voltage references coming from separate phases. But that only covers 2 phases. What about the third?

Would I have to buy a main board and 1 add on board. And split the 3 phases between the two boards. (making sure I supply separate VA per phase)

Please correct me If I am missing something.

**Comment by CircuitSetup** [comment, marked as answer]
If you are going for accuracy here is what you would need:

- 1 main board - 1 add-on board
- 3 AC transformers to bring down the voltage to 9VAC
- Cut JP12 & JP13 on the back of the main board
- Cut 2 bottom (long) pins on the add-on board (2 that the ESP32 does not plug into)
- Add connectors to the main board and add-on board for VA2-/VA2+ (these can be supplied)

This will allow you to have CT1-CT3 in phase with phase 1, CT4-CT6 in phase with phase 2, and the whole add-on board in phase with phase 3. This method isn't documented yet, but others have done this with success.

If you do not want to buy an add-on board, then you can connect the 2 phases to the main board, and estimate the voltage and power factor in software. Assuming your phases are fairly balanced, the difference in voltage shouldn't be much. If you have some kind of motor on the 3rd phase only, this could throw off the power factor. 

FYI, the blackfriday2020 coupon is valid for one more day! Hope this helps!

**Reply by sgrunf** [reply]
Hi, i noticed that ATM90E32 have 3 voltage reading channels. Why you didn't use the third voltage channel?
